### PR TITLE
Make meta function specs style-guide-compliant

### PR DESCRIPTION
**Pull request by nex3**

### CODE REVIEW

File: spec/core_functions/meta/mixin_exists.hrx [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
+<===> different_module/undefined/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> different_module/defined/input.scss
+@use "other";
+a {b: mixin-exists("c", "color")}
```
Comment (jathak) [inline spec/core_functions/meta/mixin_exists.hrx]: Should this be `mixin-exists("c", "other")`?

File: spec/core_functions/meta/mixin_exists.hrx [hunk 2]
```diff
@@ -0,0 +1,384 @@
...
+@mixin d() {}
+
+<===> different_module/chosen_prefix/output.css
+b {
+  c: true;
+}
+
+<===>
+================================================================================
+<===> different_module/through_use/options.yml
```
Comment (jathak) [inline spec/core_functions/meta/mixin_exists.hrx]: Unnecessary given the options.yml in the outer directory? (same below)

File: spec/core_functions/meta/mixin_exists.hrx (hunk 1 above)
Comment (nex3) [reply spec/core_functions/meta/mixin_exists.hrx]: It sure should!

File: spec/core_functions/meta/mixin_exists.hrx (hunk 2 above)
Comment (nex3) [reply spec/core_functions/meta/mixin_exists.hrx]: Done.

**Pull request merged**